runtime-common/stdlib/server/net-functions.cpp

```diff
@@ -0,0 +1,40 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2024 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#include "runtime-common/stdlib/server/net-functions.h"
+
+#include <arpa/inet.h>
+#include <cstdio>
+
+Optional<int64_t> f$ip2long(const string &ip) noexcept {
+  struct in_addr result;
+  if (inet_pton(AF_INET, ip.c_str(), &result) != 1) {
+    return false;
+  }
+  return ntohl(result.s_addr);
+}
+
+Optional<string> f$ip2ulong(const string &ip) noexcept {
+  struct in_addr result;
+  if (inet_pton(AF_INET, ip.c_str(), &result) != 1) {
+    return false;
+  }
+
+  const size_t buf_size = 25;
+  char buf[buf_size];
+  int len = snprintf(buf, buf_size, "%u", ntohl(result.s_addr));
+  return string(buf, len);
+}
+
+string f$long2ip(int64_t num) noexcept {
+  auto &runtime_context{RuntimeContext::get()};
+  runtime_context.static_SB.clean().reserve(100);
+  for (int i = 3; i >= 0; i--) {
```

Comment:
are we sure we don't want to use the off-the-shelf `inet_ntop` function for the conversion?